### ISSUE TITLE
Wrong use of "available presentation displays" in 6.4.4

### ISSUE THREAD

**Issue by tidoust** [w3c/presentation-api#382]
Something else I failed to notice: step 2 of the algorithm to [monitor the list of available presentation displays](http://w3c.github.io/presentation-api/#dfn-monitor-the-list-of-available-presentation-displays) instructs the user agent to "retrieve **available presentation displays**" with a link back to the definition of [available presentation display](http://w3c.github.io/presentation-api/#dfn-available-presentation-display).

However, at this stage, there is no URL against which availability can be determined, the goal is rather to retrieve a flat list of presentation displays. Next steps (3.3.1 in particular) determine whether these displays are _available_ for URLs in the set of presentation availability objects.

In other words, I think we need to drop "available" and rather link back to [presentation display](http://w3c.github.io/presentation-api/#dfn-presentation-display).

**Comment by mfoltzgoogle** [comment]
Agreed.
